server/models/Conversation.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+const Profile = require("./Profile");
+const Schema = mongoose.Schema;
+const message = new Schema({
+  sender: { type: Schema.Types.ObjectId, ref: "Profile" },
+  message: String,
+  date: Date,
+});
```

Comment:
Why not have a message model that defines a message schema in the `models` directory instead of defining more than one schema in a single file?